pandas/tests/types/test_common.py

```diff
@@ -80,3 +81,22 @@ def test_dtype_equal_strict():
     assert not is_dtype_equal(
         pandas_dtype('datetime64[ns, US/Eastern]'),
         pandas_dtype('datetime64[ns, CET]'))
+
+
+get_dtype_funcs = [
+    (com.is_dtype_equal, 2),
```

Comment:
what I meant was to add *all* of the validation functions.

```
(In [3]: [ f for f in dir(com) if f.startswith('is_') and f.endswith('dtype')]
Out[3]: 
['is_any_int_dtype',
 'is_bool_dtype',
 'is_categorical_dtype',
 'is_complex_dtype',
 'is_datetime64_any_dtype',
 'is_datetime64_dtype',
 'is_datetime64_ns_dtype',
 'is_datetime64tz_dtype',
 'is_datetime_or_timedelta_dtype',
 'is_float_dtype',
 'is_floating_dtype',
 'is_int64_dtype',
 'is_int_or_datetime_dtype',
 'is_integer_dtype',
 'is_numeric_dtype',
 'is_object_dtype',
 'is_period_dtype',
 'is_signed_integer_dtype',
 'is_string_dtype',
 'is_string_like_dtype',
 'is_timedelta64_dtype',
 'is_timedelta64_ns_dtype',
 'is_unsigned_integer_dtype']
```

then ``is_dtype_equal`` should be a separate function to test (as its different)